packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -6,7 +6,6 @@ import chaiAsPromised from 'chai-as-promised'
 import {
   DeploymentConfig,
   ConfigArtifacts,
-  Create2ActionInput,
```

Comment:
The diff in this file is a mess, so you don't need to scrutinize it. I overhauled the structure of the integration tests in this file to reduce the amount of boilerplate code, since this made it easier to write the regression test, which executes two deployments.